src/common/config/state/init.js

```diff
@@ -82,7 +82,7 @@ const model = () => {
       },
       set block_selector (val) {
         if (isValidSelector(val)) hiddenState.block_selector += `,${val}`
-        else warn('An invalid session_replay.block_selector was provided and will not be used', val)
+        else if (val !== '') warn('An invalid session_replay.block_selector was provided and will not be used', val)
```

Comment:
Still throws a warning for `null` but probably a super edge case.

Reply:
I think the group consensus was to not "allow" a null value for block selector anyway so this might be ok